test/mocha/campaigns/PR/10169.js

```diff
@@ -20,6 +22,15 @@ let productData = {
 
 scenario('PR-10169: Check that the images is well displayed in the edit combination', () => {
   authentication.signInBO('10169');
+  scenario('Close the onboarding modal if exist ', client => {
```

Comment:
You can add a function to close the onboarding modal, and you can use it for all files